scripts/github-repos/README.md

```diff
@@ -1,53 +1,138 @@
-# Bulk creation/deletion of many repos and cs169a-team
+# Bulk creation/deletion of many repos and student teams
 
 ```
-Usage: ./github-repos.rb [required options] [invite|repos|remove|remove_access]
+Usage: ./github-repos.rb [required options] [invite|create_teams|indiv_repos|team_repos|remove_indivi_repos|remove_team_repos|remove_teams|remove_team_repo_access] [optional options]
 
 GITHUB_ORG_API_KEY for the org must be set as an environment variable.
 
-'invite' invites students provided in .csv file and creates teams, 'repos' creates team repos, 'remove' remove students, repos, teams from the org
-
 It's safe to run multiple times.
 
-Required arguments:
-    -c, --csv=CSVFILE                CSV file containing at least "Team" and "Email" named columns
-    -o, --orgname=ORGNAME            The name of the org eg org_name
-    -f, --filename=FILENAME          The base filename for repos, eg "fa23-actionmap-04", actionmap is the base file name of the repo
-    -p, --prefix=PREFIX              Semester prefix, eg "fa23" create a repos prefix, "fa23-actionmap-04", etc.
-    -t, --template=TEMPLATE          The repo name within the org to use as template eg repo_name (Assume the repo own by org)
+'invite':
+Create a team called STUDENTTEAM under the org that has all students in the same semester and send invitation
+If STUDENTTEAM is already exist, the script will resent invitation to students. 
+(Temporary for special situations in Fa23)

```

Comment:
why is this temporary? isn't this always the desired behavior?  i really don't want to have changes in a script that are there for one semester only.  let's discuss what special case is being addressed here and whether it can be addressed without creating a change now that will HAVE to be undone after the semester, since the rest of us have to live with these scripts after you're gone....

Reply:
Sorry for the confusion here. The temporary means the optional prefix option. In the future, I believe we will not have child teams at the first few weeks. Child teams are for chip10.5 only. So, the invite command should invite students to the STUDENTTEAM rather than create child teams and invite them to those child teams.

If the prefix option is provided, the scripts will assume there is 'Team' column in csv file, creates child teams, and invite students to child teams.